record-and-playback/core/scripts/post_events/post_events_analytics.rb.example

```diff
@@ -0,0 +1,203 @@
+    if item.start_with?("gl_")
+      tmp_metadata = tmp_metadata.tap { |hs|
+        hs.delete(item) 
+      }
+    end
+  }
+
+  # Remove meeting id as we don't want it passed to 3rd party inside metadata
+  tmp_metadata.delete("meeting_id")
+
+  # Remove the internal meeting id generated by bbbevents gem
+  data = data.tap { |hs|
+    hs.delete("meeting_id") 
+  }
+
+  # Convert CamelCase keys to snake_keys. This is done in bbbevents gem
+  # but we do it here too anyways.
+  tmp_metadata.deep_transform_keys! do |key|
+    k = key.to_s.underscore rescue key
+    k.to_sym rescue key
+  end       
+
+  # Remove all internal user ids
+  attendees = data["attendees"]
+  attendees.each { |attendee|
+    attendee.delete("id")
+  }
+
+end
+
+opts = Trollop::options do
+  opt :meeting_id, "Meeting id to archive", :type => String
+end
+meeting_id = opts[:meeting_id]
+
+i_got_this = false
+if not i_got_this
+  BigBlueButton.logger.info("######################################################")
+  BigBlueButton.logger.info("!!! Please read setup instructions in bottome of this script. !!!")
+  BigBlueButton.logger.info("######################################################")
+  exit 0
+end
+
+
+logger = Logger.new(STDOUT)
+#logger = Logger.new("/var/log/bigbluebutton/post_events.log", 'weekly' )
+logger.level = Logger::INFO
+BigBlueButton.logger = logger
+
+archived_files = "/var/bigbluebutton/recording/raw/#{meeting_id}"
+meeting_metadata = BigBlueButton::Events.get_meeting_metadata("#{archived_files}/events.xml")
+
+props = YAML::load(File.open('bigbluebutton.yml'))
+
+events_xml_path = "#{archived_files}/events.xml"
+data_json_path = "#{archived_files}/data.json"
+
+if meeting_metadata["analytics_callback_url"] != nil
+  analytics_callback_url = meeting_metadata["analytics_callback_url"]
+
+  # Parse the events.xml.
+  events_data = BBBEvents.parse(events_xml_path)
+
+  # Write JSON data to file.
+  File.open(data_json_path, 'w') do |f|
+    f.write(events_data.to_json)
+  end  
+
+  json_file = File.open data_json_path
+  data = JSON.load(json_file)
+
+  format_analytics_data!(data)
+
+  # Get the external meeting id
+  external_meeting_id = meeting_metadata["meetingId"]
+
+  data_version_format = "1.0"
+
+  payload = { version: data_version_format,
+    ext_meeting_id: external_meeting_id, 
+    meeting_id: meeting_id,
+    data: data
+  }
+
+  # Convert CamelCase keys to snake_keys for the whole payload.
+  # This is a sledgehammer to force keys to be consistent.
+  payload.deep_transform_keys! do |key|
+    k = key.to_s.underscore rescue key
+    k.to_sym rescue key
+  end
+
+  #puts payload.to_json
+
+  secret = "my_secret" #or the shared secret from bbb.properties
+  send_data(analytics_callback_url, secret, payload)
+end
+
+
+
+exit 0
+
+
+##################
+#### Setup instructions
+# Modify core/Gemfile by adding
+#
+# gem "jwt"
+# gem 'bbbevents', git: 'https://github.com/ritzalam/bbb-events.git', branch: 'v2-dev'
```

Comment:
I suggest you to move bbb-events to the bigbluebutton organization, and move this statement to `/usr/local/bigbluebutton/core/Gemfile` (as well as Gemfile.lock).